maestro-xcuitest-driver/src/main/kotlin/xcuitest/installer/LocalXCTestInstaller.kt

```diff
@@ -26,29 +26,36 @@ class LocalXCTestInstaller(
 
     private var xcTestProcess: Process? = null
 
-    override fun killAndUninstall() {
+    override fun uninstall() {
```

Comment:
This doesn't aligns with what function is doing, I see that its killing a process as well. We might want to call `stop` with `uninstall` wherever its needed or rename this.

Reply:
uninstalling implies also stopping the process

Reply:
Ummm, uninstalling gave me the impression that it's just removing the app. That's why I was being skeptical about the name.